addons/im_livechat/controllers/main.py

```diff
@@ -222,6 +222,7 @@ def feedback(self, channel_id, rate, reason=None, **kwargs):
                 'consumed': True,
                 'feedback': reason,
                 'is_internal': False,
+                'rated_on': fields.Datetime.now(),
```

Comment:
Rather than changing all occurrences of rating record 
We should try to update values in the write/create the method of rating
Let me know if that works for you